README.md

```diff
@@ -148,6 +148,30 @@ spec:
 Please be aware that the taint and toleration only ensures that no other pod gets scheduled to a PostgreSQL node 
 but not that PostgreSQL pods are placed on such a node. This can be achieved by setting a node affinity rule in the ConfigMap.
 
+### Using the operator to minimize the amount of failovers during the cluster upgrade
+
+To make sure there is only one failover per each PostgreSQL deployment when nodes of the cluster are decomissioned
```

Comment:
I feel the explanation from Jan is worth adding here:

> We use this readiness label during migrations of the underlying nodes to avoid scheduling pods/masters on nodes that will also be recycled

Reply:
Well, this is right below
`when a postgres pod is killed by the operator because the node is decomissioned, it will not respawn on another node to-be-decomissione`